packages/server/src/api/controllers/query/index.ts

```diff
@@ -255,7 +255,7 @@ async function execute(
     if (opts && opts.rowsOnly) {
       ctx.body = rows
     } else {
-      ctx.body = { data: rows, pagination, ...extra }
+      ctx.body = { data: rows, pagination, ...extra, ...info }
```

Comment:
is there any potential for this to override things from `pagination` and `extra`? Should we namespace the response info?

Reply:
Had a look and didn't see any conflicts for `code`, `time` and `size`